src/components/Calculator.js

```diff
@@ -1,46 +1,82 @@
+import PropTypes from 'prop-types';
 import React from 'react';
+import calculate from '../logic/calculate';
 import './Calculator.css';
 
+function DisplayTotal(props) {
+  const { total, next, operation } = props;
+  return (
+    <div className="display">
+      <h1>
+        {total}
+        {operation}
+        {next}
+      </h1>
+    </div>
+  );
+}
+
+DisplayTotal.propTypes = {
+  total: PropTypes.string.isRequired,
+  next: PropTypes.string.isRequired,
+  operation: PropTypes.string.isRequired,
+};
+
```

Comment:
- [ ] (Optional) It would be nice to create a file and put this component in it. Then import it into your _**Calculator.js**_ file.